sadu-queries/src/main/java/de/chojo/sadu/wrapper/util/Row.java

```diff
@@ -77,6 +78,51 @@ public String getString(int columnIndex) throws SQLException {
         return resultSet.getString(columnIndex);
     }
 
+    /**
+     * Retrieves the value of the designated column in the current row
+     * of this {@code Row} object as
+     * a {@code String} in the Java programming language and maps it to an enum.
+     *
+     * @param columnIndex the first column is 1, the second is 2, ...
+     * @param clazz       the enum clazz
+     * @param <T>         type of enum
+     * @return the column value; if the value is SQL {@code NULL}, the
+     * value returned is {@code null}
+     * @throws SQLException             if the columnIndex is not valid;
+     *                                  if a database access error occurs or this method is
+     *                                  called on a closed result set
+     * @throws IllegalArgumentException when the value could not be mapped
+     * @throws NullPointerException     when the class is null
+     *
+     * @since 1.2
+     */
+    public <T extends Enum<?>> T getEnum(int columnIndex, Class<T> clazz) throws SQLException {
+        return EnumConversion.fromString(getString(columnIndex), clazz, false);
```

Comment:
Also you can get rid of the `EnumConversion` class and just use the suggested method here directly.